sydney/constants.py

```diff
@@ -1,13 +1,21 @@
-HEADERS = {
+CHAT_HEADERS = {
     "Accept": "application/json",
     "Accept-Encoding": "gzip, deflate",
     "Accept-Language": "en-US,en;q=0.9",
     "Content-Type": "application/json",
     "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/117.0.0.0 Safari/537.36 Edg/117.0.2045.47",
 }
+KBLOB_HEADERS = {
+    "Accept-Language": "en-US,en;q=0.5",
+    "Content-Type": "multipart/form-data",
+    "Referer": "https://www.bing.com/",
+    "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36 Edg/112.0.1722.48"
```

Comment:
It's a minor thing, but for consistency, it might be best to use the same `User-Agent` as above.